{
  "throws": "Can not use 'await' as identifier inside an async function (1:20)"
}